integration_tests/commands/resp/hexists_test.go

```diff
@@ -0,0 +1,73 @@
+package resp
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestHExists(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+
+	testCases := []TestCase{
+		{
+			name:     "RESP wrong number of arguments for HEXISTS",
+			commands: []string{"HEXISTS", "HEXISTS KEY", "HEXISTS KEY FIELD ANOTHER_FIELD"},
+			expected: []interface{}{"ERR wrong number of arguments for 'hexists' command",
+				"ERR wrong number of arguments for 'hexists' command",
+				"ERR wrong number of arguments for 'hexists' command"},
+		},
+		{
+			name:     "RESP HEXISTS non existent key",
+			commands: []string{"HSET key_hExists1 field value", "HEXISTS wrong_key_hExists field"},
+			expected: []interface{}{int64(1), int64(0)},
+		},
+		{
+			name:     "RESP HEXISTS non existent field",
+			commands: []string{"HSET key_hExists2 field value", "HEXISTS key_hExists2 wrong_field"},
+			expected: []interface{}{int64(1), int64(0)},
+		},
+		{
+			name:     "RESP HEXISTS existent key and field",
+			commands: []string{"HSET key_hExists3 field HelloWorld", "HEXISTS key_hExists3 field"},
+			expected: []interface{}{int64(1), int64(1)},
+		},
+		{
+			name:     "RESP Check if field exists when k f and v are set",
+			commands: []string{"HSET key field value", "HEXISTS key field"},
+			expected: []interface{}{int64(1), int64(1)},
+			debug:    true,
+		},
+		{
+			name:     "RESP Check if field exists when k exists but not f and v",
+			commands: []string{"HSET key field1 value", "HEXISTS key field"},
+			expected: []interface{}{int64(1), int64(0)},
+			debug:    true,
+		},
+		{
+			name:     "RESP Check if field exists when no k,f and v exist",
+			commands: []string{"HEXISTS key field"},
+			expected: []interface{}{int64(0)},
+			debug:    true,
+		},
+		{
+			name:     "RESP HEXISTS operation against a key holding the wrong kind of value",
+			commands: []string{"SET key value", "HEXISTS key field"},
+			expected: []interface{}{"OK", "ERR -WRONGTYPE Operation against a key holding the wrong kind of value"},
```

Comment:
Please confirm on this error message in redis for same test case

Reply:
![image](https://github.com/user-attachments/assets/46a27664-e8ac-4e96-87ab-55669cb73f1f)


This is the error message, the texts match.